angular-ngrx-scss/src/app/home/home.module.ts

```diff
@@ -35,6 +36,7 @@ import { OrganizationInfoComponent } from './profile/org-profile/org-info/org-in
     UserProfileComponent,
     OrgProfileComponent,
     OrganizationInfoComponent,
+    RepoFilterDropdownComponent,
```

Comment:
If you rebase off main branch now, there is a shared module created. Is it better to import and reexport this component there?

Reply:
yes, let me rebase and import form there.